playground/milvus/MilvusPlayground.AppHost/Properties/launchSettings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "https": {
+      "commandName": "Project",
+      "dotnetRunMessages": true,
+      "launchBrowser": true,
+      "applicationUrl": "https://localhost:17115;http://localhost:15293",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "DOTNET_ENVIRONMENT": "Development",
+        "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL": "https://localhost:21286",
+        "DOTNET_RESOURCE_SERVICE_ENDPOINT_URL": "https://localhost:22004"
+      }
+    },
+    "http": {
```

Comment:
`generate-manifest` profile here instead of where I said above :)

Reply:
Will add this, but curious what's the point for doing so in the playground source here? Convenient launching point to generate the manifest?